apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -3,9 +3,10 @@ import { Theme, withStyles } from '@material-ui/core/styles';
 import { ClassNameMap, Styles } from '@material-ui/core/styles/withStyles';
 import { Delete, Search } from '@material-ui/icons';
 import { WebsocSectionFinalExam } from '@packages/antalmanac-types';
-import { useEffect, useRef, useCallback } from 'react';
+import { useEffect, useRef } from 'react';
 import { Event } from 'react-big-calendar';
-import { Link } from 'react-router-dom';
+
+import { MapLink } from '../buttons/MapLink';
```

Comment:
```suggestion
import { MapLink } from '$components/buttons/MapLink';
```
issue: prefer import aliases whenever possible, as this makes refactoring (and readability) better, compared to relative imports